CI rework - use kind, validate->test->publish, contrib and release rework (#1422)
jupyterhub/zero-to-jupyterhub-k8s: tests/README.md

# What is this folder about?

> We have setup tests for [pytest](https://docs.pytest.org/en/latest/) that will run in our CI/CD pipeline on Travis. These test assumes it is able to speak directly to a running hub within a Kubernetes cluster etc. In practice, they assume you have been using `dev` script to set it all up.

Reviewer: ```suggestion
This folder contains tests that [pytest](https://docs.pytest.org/en/latest/) will run in our CI/CD pipeline on Travis. These tests must be able to speak directly to a running hub within a Kubernetes cluster etc. In practice, the tests will be set up using a `dev` script.
```
